bin/citrea/tests/bitcoin/guest_cycles.rs

```diff
@@ -150,6 +148,7 @@ async fn guest_cycles() {
     env::set_var("RISC0_DEV_MODE", "1");
     env::set_var("RISC0_INFO", "1");
     env::set_var("RUST_LOG", "info");
+    env::set_var("RISC0_PPROF_OUT", "profile.pb");
```

Comment:
Is it required?

Reply:
it is needed in guest cycles test. didnt want to open another pr just for this one line.